new-dti-website-redesign/src/components/FAQAccordion.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { ReactNode } from 'react';
+
+// Props type for the FAQAccordion component
+type FAQAccordionProps = {
+  header: string;
+  children: ReactNode;
+};
+
+/**
+ * Creates a collapsible accordion component using Radix UI
+ *
+ * @param header - The text displayed in the accordion trigger/header
+ * @param children - The content shown when the accordion is opened
+ */
+export default function FAQAccordion({ header, children }: FAQAccordionProps) {
+  return (
+    <details className="group border border-border-1">
+      <summary
+        className="
+          list-none p-8
+          focus-visible:outline focus-visible:outline-[2px] focus-visible:outline-offset-[3px] focus-visible:outline-[var(--foreground-1)] focus-visible:relative 
+          group-open:pb-[16px] bg-background-1 group-open:bg-background-2 hover:bg-background-2 group-open:hover:bg-background-3
```

Comment:
```suggestion
          group-open:pb-[16px] bg-background-1 group-open:bg-background-2 hover:bg-background-2 group-open:hover:bg-background-3 transition-[background-color] duration-[120ms]
```

Just to add a lil transition (also consistent with other interactive elements we have)